### ISSUE TITLE
Various HTML issues

### ISSUE THREAD

**Issue by corrideat** [okTurtles/group-income#1435]
### Problem

1. The app sets `lang="eng"`, which is an invalid language.
2. IE conditional comments are used. Conditional comments have been removed since IE 10, released in 2012 (that's over 10 years ago). The last version of IE (IE 11) was released in 2013, and Microsoft have announced its end of life. It's unlikely that someone will be browsing the app using such an outdated browser.
3. Related to IE: meta x-ua-compatible is set.

### Solution

1. Set `lang="en"` instead
2. Remove conditional comments. If showing a warning for unsupported / outdated browsers is desired, this should be some additional JavaScript that does it. The reasoning is that the (1) app doesn't work at all in IE 10 or IE 11 (with no warning shown), (2) it's far more likely that someone will use IE 11 than IE <10, and it seems odd to show the warning only to these people using incredibly outdated systems (which probably won't even be able to connect to the app in the first place, as the latest OSes to support these don't support modern TLS) and (3) it'd be nice to show such a warning to users of other outdated browsers (e.g., old FF versions)
3. Remove x-ua-compatible.

**Comment by taoeffect** [comment]
Closed!
